xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/menus_macros.vm

```diff
@@ -224,12 +224,23 @@
  * Display extension points only if the author of the UIX has the admin right on the wiki.
  *
  * @param $name name of the extension points to display
+ * @param $revert set to get the uixs in revert order
  * @since 7.3RC1
  *#
-#macro(displaySimpleSecureUIX $name)
-  #foreach($uix in $services.uix.getExtensions($name, {'sortByParameter' : 'order'}))
-    #if ($services.security.authorization.hasAccess('admin', $uix.authorReference, $services.wiki.currentWikiDescriptor.reference))
-      $services.rendering.render($uix.execute(), 'html/5.0')
+#macro(displaySimpleSecureUIX $name $revert)
+  #set ($uixs = $services.uix.getExtensions($name, {'sortByParameter' : 'order'}))
+  #set ($numberOfUixs = ${uixs.size()} - 1)
+  #if ($numberOfUixs > 0)
+    #foreach($uixID in [0..$numberOfUixs])
```

Comment:
Not a huge fan of this loop, it sure is working, but I might have used some $collectiontool helper to get a reverted list instead of playing with the indexes. Not blocking the merge though.